test/ui-test/testSuites/suite_status/shared/scripts/sections/community_names.py

```diff
@@ -8,6 +8,9 @@
 mainWindow_createChannelOrCategoryBtn_StatusBaseText = {"container": statusDesktop_mainWindow, "objectName": "createChannelOrCategoryBtn", "type": "StatusBaseText", "visible": True}
 create_channel_StatusMenuItemDelegate = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "enabled": True, "objectName": "createCommunityChannelBtn", "type": "StatusMenuItemDelegate", "visible": True}
 create_category_StatusMenuItemDelegate = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "enabled": True, "objectName": "createCommunityCategoryBtn", "type": "StatusMenuItemDelegate", "visible": True}
+edit_сategory_StatusMenuItemDelegate = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "enabled": True, "objectName": "editCategoryMenuItem", "type": "StatusMenuItemDelegate", "visible": True}
```

Comment:
Is `checkable` property mandatory here?

Reply:
don't think so, i just trying to keep code close other cases. It's also not necessary for categories

Reply:
the names files are still a bit messy. The clean ups helped a lot, but they can still be improved,  so don,t take anything you see in those files as perfect practices :sweat_smile: 
Anyway, if we don't need `checkable`, better to remove it, that way, there is less chance for the name to break if the UI changes